samples/android/tutorial-3-cameracontrol/build.gradle.in

```diff
@@ -29,8 +29,10 @@ android {
 dependencies {
     //implementation fileTree(dir: 'libs', include: ['*.jar'])
     if (gradle.opencv_source == "sdk_path") {
+        println 'Consider OpenCV from SDK'
```

Comment:
I like "Using OpenCV from ..." line used in `tutorial-2-mixedprocessing/jni/CMakeLists.txt` more :slightly_smiling_face: